_posts/2019-01-23-cockpit-186.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Cockpit 186
+author: Katerina Koukiou
+date: 2019-01-23
+tags: cockpit linux
+slug: cockpit-186
+category: release
+summary: Responsive system page and improvement in services page
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We
+release regularly. Here are the release notes from version 186.
+
+### Services: Redesign Services page
+
+During usability testing we found out that the service details page was hard to
+understand for administrators with a non-Linux background. The Services page got
+updated to use more neutral and well-known terminology and a simplified layout.
+
+![services screenshot](/images/services-relayout.png)
+
+### Responsive System Overview page
+
+These UI elements now adjust their layout to small and mobile browser
+windows. This improvement will come to more pages in the near
+future.
```

Comment:
The overview page now more optimally adjusts to small window sizes and mobile browsers. This is part of an ongoing process to improve layout across all of Cockpit.

Reply:
"more optimally" isn't really a thing :wink: "adjusts better to small..."?